codegen/src/test/java/software/amazon/awssdk/codegen/naming/DefaultNamingStrategyTest.java

```diff
@@ -16,12 +16,35 @@
 package software.amazon.awssdk.codegen.naming;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
+import java.util.ArrayList;
+import java.util.Map;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import software.amazon.awssdk.codegen.model.service.Member;
+import software.amazon.awssdk.codegen.model.service.ServiceModel;
+import software.amazon.awssdk.codegen.model.service.Shape;
 
+@RunWith(MockitoJUnitRunner.class)
 public class DefaultNamingStrategyTest {
 
-    private DefaultNamingStrategy strat = new DefaultNamingStrategy(null, null);
+    private ServiceModel serviceModel = mock(ServiceModel.class);
+
+    @Mock
+    private Map<String, Shape> mockShapeMap;
+
+    @Mock
+    private Shape mockShape;
+
+    @Mock
+    private Member member;
```

Comment:
Why do these need to be mocked? Can we just created actual instances and set the appropriate members?

Reply:
It was easy to just mock the methods I need in these classes instead of creating new object every time.